Mathlib/Algebra/GroupWithZero/NonZeroDivisors.lean

```diff
@@ -259,11 +259,7 @@ lemma mul_mem_nonZeroDivisors : a * b ∈ M₀⁰ ↔ a ∈ M₀⁰ ∧ b ∈ M
     constructor <;> intro x h' <;> apply h
     · rw [← mul_assoc, h', zero_mul]
     · rw [mul_comm a b, ← mul_assoc, h', zero_mul]
-  mpr := by
-    rintro ⟨ha, hb⟩ x hx
-    apply ha
-    apply hb
-    rw [mul_assoc, hx]
+  mpr := by aesop
```

Comment:
You golfed this line differently in the other big golfing PR. I guess I prefer the other proof.